eng/docs/index/Generate-DocIndex.ps1

```diff
@@ -24,22 +25,16 @@ Copy-Item "${DocGenDir}/templates/*" -Destination "${DocOutDir}/templates" -Forc
 Copy-Item "${DocGenDir}/docfx.json" -Destination "${DocOutDir}/" -Force
 
 Write-Verbose "Creating Index using service directory and package names from repo..."
-$ServiceList = Get-ChildItem "$($RepoRoot)/sdk" -Directory -Exclude eng, mgmtcommon, template | Sort-Object
+$ServiceList = Get-ChildItem "$($RepoRoot)/sdk" -Directory -Exclude eng, mgmtcommon | Sort-Object
```

Comment:
why is eng and mgmtcommon excluded here? they don't look to be under the sdk folder.

Reply:
This was brought over from .NET. They aren't in those directories... I should also add template back in here.